webtool/ytdlp_update.py

```
@@ -624,12 +640,25 @@ def _venv_kennung() -> str:
+    naechsten Start auf einen anderen Namen. `normcase`, weil Windows-Pfade
+    gross-/kleinschreibungsblind sind.
+
+    **`realpath` statt `abspath`, und seit #254 ist das kein Feinschliff mehr.** Solange die
+    Kennung nur den MERKER trug, war die Folge zweier Schreibweisen derselben venv eine
+    verpasste Reparatur — so stand es hier, und so stimmte es. Seit die Kennung die SPERRE
+    traegt, ist die Folge eine andere: zwei Sperren fuer dasselbe `site-packages`, also zwei
+    gleichzeitige `pip install` hinein — genau der Schaden, gegen den es die Sperre gibt. Vor
+    #254 war der Fall gedeckt (EINE Sperre je Nutzer deckte jede Schreibweise mit ab); das ist
+    die ehrliche Antwort auf „was erlaubt der Fix NEU?", und sie kostet ein Wort.
+    **Gemessen, nicht angenommen** — `abspath` gegen `realpath` an denselben Zielen:
+    Junction (`New-Item -ItemType Junction`) → `…\jj` gegen `…\echt`; `subst`-Laufwerk →
```

Ruff (0.16.1)

[warning] Invalid escape sequence: `\j`

Use a raw string literal

(W605)